impl/cluster/src/main/java/org/pentaho/big/data/impl/cluster/NamedClusterManager.java

```diff
@@ -372,7 +373,8 @@ private String getSlaveServerMetastoreDir() throws IOException {
       // see if metastore was copied to the big data plugin folder (yarn kettle cluster job)
       slaveMetaStorePath = pluginInterface.getPluginDirectory().getPath();
       slaveMetastoreDir =
-        KettleVFS.getFileObject( slaveMetaStorePath + File.separator + XmlUtil.META_FOLDER_NAME );
+        KettleVFS.getInstance( DefaultBowl.getInstance() )
```

Comment:
why the default bowl here?

Reply:
Hadoop will be covered in https://hv-eng.atlassian.net/browse/BACKLOG-40206